src/header/searchResults/searchResults.ts

```diff
@@ -210,6 +210,7 @@ export class SearchResults extends DefaultControl {
     const options = {
       floor: isFinite(mwzObject.floor) ? translate('on_floor', { floor: mwzObject.floor }) : '',
       icon: getIcon(mwzObject),
+      placeList: has(mwzObject, 'placeIds') ? translate('number_of_places', { placeList: mwzObject.placeIds.length }) : '',
```

Comment:
Something like `places_count` can be better than `placeList`
```suggestion
      places_count: has(mwzObject, 'placeIds') ? translate('number_of_places', { placeList: mwzObject.placeIds.length }) : '',
```

Reply:
Can you take a look about empty placeLists?
Empty placeLists must not be displayed in results